tests/e2e/util.go

```diff
@@ -2348,6 +2347,26 @@ func deleteFcdWithRetriesForSpecificErr(ctx context.Context, fcdID string, dsRef
 	return waitErr
 }
 
+//updateDeploymentImage updates the image in the deployment with the given image
+func updateDeploymentImage(ns string, image string, deploymentName string, imageName string) {
```

Comment:
Please rework on this method.

You can use `UpdateDeploymentWithRetries()` and `WaitForDeploymentCompleteAndCheckRolling()`